Source/WebCore/accessibility/AccessibilityObject.h

```diff
@@ -247,6 +249,7 @@ class AccessibilityObject : public AXCoreObject {
     Node* node() const override { return nullptr; }
     RenderObject* renderer() const override { return nullptr; }
     RenderObject* rendererOrNearestAncestor() const;
+    bool isContinuation() const;
```

Comment:
I feel like somewhere we need a comment explaining what this is and why it matters. The next person who joins the team won't know what a continuation is.

What if we called it nodeMayHaveMultipleRenderers() instead, where "is continuation" is the implementation detail, or maybe that could be used for the YouTube embed?